recommender/errors.py

```diff
@@ -20,6 +25,11 @@ def message(self):  # pragma: no cover
         )
 
 
+class UntrainedRLAgentError(RecommendationEngineError):
+    def message(self):  # pragma: no cover
+        return "RL-Agent can't operate without trained actor models"
```

Comment:
same as above

Reply:
Now, this is plural (model**s**), so no "a" before "trained"